packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -24,23 +24,13 @@ export const DEFAULT_NAME = 'Untitled SQL Query';
 export interface SqlRunnerState {
     projectUuid: string;
     activeTable: string | undefined;
-    savedSqlUuid: string | undefined;
-    slug: string | undefined;
-    space:
-        | {
-              uuid: string;
-              name: string;
-          }
-        | undefined;
+    savedSqlChart: SqlChart | undefined;
```

Comment:
can we then do this?
```suggestion
    readonly savedSqlChart: SqlChart | undefined;
```

Reply:
Ignore ^ it has to be set anyway